tests/scripts/arp_responder.py

```diff
@@ -7,7 +7,12 @@
 import os.path
 from fcntl import ioctl
 from pprint import pprint
-
+import logging
+logging.getLogger("scapy.runtime").setLevel(logging.ERROR)
+import ptf.packet as scapy
+import scapy.all as scapy2
```

Comment:
We could probably be a little clearer with the naming here, something like `packet` and `scapy`?

Reply:
the file is copied, let's create backlog item for this and the likes below